{"name": "test_UserLogin_ddt_006", "status": "broken", "statusDetails": {"message": "TypeError: 'NoneType' object is not iterable", "trace": "self = <testCases.test_UserProfile_DDT.Test_User_Profile_DDT object at 0x000001F7259BDA10>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"aaab1a8c0fd268f0b543b213656a7b64\")>\n\n    def test_UserLogin_ddt_006(self, setup):\n        self.driver = setup\n    \n        self.ur = UserProfile_Class(self.driver)\n        self.rows = ExcelFileOperation.rows_count(self.path, 'Sheet1')\n        print(self.rows)\n        Result_List = []\n        for r in range(2, self.rows + 1):\n            self.email = ExcelFileOperation.ReadData(self.path, 'Sheet1', r, 1)\n            self.password = ExcelFileOperation.ReadData(self.path, 'Sheet1', r, 2)\n            self.Exp_Result = ExcelFileOperation.ReadData(self.path, 'Sheet1', r, 3)\n            self.driver.get(self.LoginUrl)\n>           self.ur.Enter_Email(self.email)\n\ntestCases\\test_UserProfile_DDT.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npageObjects\\UserProfilePage.py:19: in Enter_Email\n    self.driver.find_element(*UserProfile_Class.Text_Email_ID).send_keys(email)\n..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py:232: in send_keys\n    Command.SEND_KEYS_TO_ELEMENT, {\"text\": \"\".join(keys_to_typing(value)), \"value\": keys_to_typing(value)}\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nvalue = (None,)\n\n    def keys_to_typing(value: Iterable[AnyKey]) -> List[str]:\n        \"\"\"Processes the values that will be typed in the element.\"\"\"\n        characters: List[str] = []\n        for val in value:\n            if isinstance(val, Keys):\n                # Todo: Does this even work?\n                characters.append(val)\n            elif isinstance(val, (int, float)):\n                characters.extend(str(val))\n            else:\n>               characters.extend(val)\nE               TypeError: 'NoneType' object is not iterable\n\n..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py:137: TypeError"}, "attachments": [{"name": "stdout", "source": "7fbf70ab-77c3-4a85-8db2-ebe762dfa6ce-attachment.txt", "type": "text/plain"}], "start": 1705084478771, "stop": 1705084491594, "uuid": "26e903b0-19a0-4507-a5c4-ef7dc2b4d12f", "historyId": "45b493c128820c8e3fd1f2a2b0a96b86", "testCaseId": "cdcd9f5e7760dfcd20ce9923e0cb836e", "fullName": "testCases.test_UserProfile_DDT.Test_User_Profile_DDT#test_UserLogin_ddt_006", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_UserProfile_DDT"}, {"name": "subSuite", "value": "Test_User_Profile_DDT"}, {"name": "host", "value": "KISHOR-8888PC"}, {"name": "thread", "value": "25524-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_UserProfile_DDT"}]}